ansible_base/resource_registry/serializers.py

```diff
@@ -98,9 +102,12 @@ def create(self, validated_data):
             if not validated_data["resource_type"]:
                 raise serializers.ValidationError({"resource_type": _("This field is required for resource creation.")})
 
-            r_type = ResourceType.objects.get(name=validated_data["resource_type"])
+            resource_type = ResourceType.objects.get(name=validated_data["resource_type"])
+            if not resource_type.can_be_managed:
+                raise serializers.ValidationError({"resource_type": _(f"Resource type: {resource_type.name} cannot be managed by Resources.")})
```

Comment:
The other change here (out of essentially 2 major changes) is to move the "managed" validation from the models to the serializer. Is that a correct summary?

I assume so that some internal thing can modify stuff.

Reply:
The gateway needs to request resource information from the resources API on controller and then create a copy of that resource internally using the Resource model.

We actually may want to get rid of this flag entirely and just control this with a flag on the jwt.